app/App.jsx

```diff
@@ -20,12 +21,13 @@ export default function App() {
         initialRouteName="Login"
```

Comment:
Resolved a merge conflict here. Remember to change initialRouteName back to login before sending a PR